launch.go

```diff
@@ -88,6 +93,23 @@ func createWorkspace(srcPaths ...string) (Workspace, error) {
 	return w, nil
 }
 
+// checkPR checks to see if the job name is a pull request and returns its number
+func checkIfPR(jobName string) (string, error) {
+	prNumber := ""
+	reg := "^PR-[0-9]+$"
+
+	matched, err := matchString(reg, jobName)
+	if err != nil {
+		return prNumber, nil
+	}
+
+	if matched == true {
+		prNumber = strings.Split(jobName, "-")[1]
+	}
```

Comment:
definitely no need to mock out strings.Split. You can always control what gets sent to it